data/templates/dhcp-server/dhcpdv6.conf.j2

```diff
@@ -11,6 +11,13 @@ option dhcp6.preference {{ preference }};
 {% if global_parameters.name_server is vyos_defined %}
 option dhcp6.name-servers {{ global_parameters.name_server | join(', ') }};
 {% endif %}
+{% if global_parameters.cisco_voip.length_width is vyos_defined and global_parameters.cisco_voip.width is vyos_defined  %}
+option space ciscovoip code width {{ global_parameters.cisco_voip.width }} length width {{ global_parameters.cisco_voip.length_width }};
```

Comment:
It feels like length is a fixed value? If this is true we do not need it on the CLI. Same for width.

Reply:
Ad I understand It is codes that can be configured